source/world/level/Level.cpp

```diff
@@ -31,13 +31,13 @@ Level::Level(LevelStorage* pStor, const std::string& str, int32_t seed, int stor
 	field_B08 = 0;
 	field_B0C = 0;
 
-	m_random.setSeed(1); // initialize with a seed of 1
+	m_random.setSeed(1); // initialize with tilePos1_x seed of 1
```

Comment:
I think Visual Studio's built-in rename feature went rouge on some of the comments. I found a few others where fields in comments outside of the intended class were incorrectly renamed. So none of the actual code should be affected by this. This will be fixed in an upcoming commit.